### DISCUSSION TITLE
更新异常

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4249] [Troubleshooting | 故障排查]
正在设置 dde-api (6.0.6) ...
-sh: 1: .: cannot open /var/lib/deepin-sound-player/.go/env: No such file
dpkg: 处理软件包 dde-api (--configure)时出错：
已安装 dde-api 软件包 post-installation 脚本 子进程返回错误状态 2
dpkg: 依赖关系问题使得 startdde 的配置工作不能继续：
startdde 依赖于 dde-api；然而：
软件包 dde-api 尚未配置。

dpkg: 处理软件包 startdde (--configure)时出错：
依赖关系问题 - 仍未被配置
dpkg: 依赖关系问题使得 deepin-desktop-environment-core 的配置工作不能继续：
deepin-desktop-environment-core 依赖于 startdde；然而：
软件包 startdde 尚未配置。

dpkg: 处理软件包 deepin-desktop-environment-core (--configure)时出错：
依赖关系问题 - 仍未被配置
dpkg: 依赖关系问题使得 dde-session-ui 的配置工作不能继续：
dde-session-ui 依赖于 startdde (>= 5.8.15)；然而：
软件包 startdde 尚未配置。

dpkg: 处理软件包 dde-session-ui (--configure)时出错：
依赖关系问题 - 仍未被配置
dpkg: 依赖关系问题使得 dde-dock 的配置工作不能继续：
dde-dock 依赖于 startdde (>= 5.8.9)；然而：
软件包 startdde 尚未配置。

dpkg: 处理软件包 dde-dock (--configure)时出错：
依赖关系问题 - 仍未被配置
dpkg: 依赖关系问题使得 dde-session 的配置工作不能继续：
dde-session 依赖于 startdde (>= 6.0.4)；然而：
软件包 startdde 尚未配置。

dpkg: 处理软件包 dde-session (--configure)时出错：
依赖关系问题 - 仍未被配置
https://bbs.deepin.org/phone/post/255706

**Comment by Decodetalkers** [comment]
`sudo dpkg --configure -a`看看呢，我看了下post脚本，它应该没有去读这个文件啊

**Reply by Linux-qitong** [reply]
我只是协助坛友转到github。
执行不了啊。

**Reply by Decodetalkers** [reply]
好吧

**Reply by Linux-qitong** [reply]
这github有时候时好时坏，这反应渠道，今天好多了，往天带的浏览器都卡（手机）。
真的烦

**Reply by Decodetalkers** [reply]
> 这github有时候时好时坏，这反应渠道，今天好多了，往天带的浏览器都卡（手机）。 真的烦

最近dns污染比较严重。
